### PR TITLE
Compatibility with mobility gem

### PR DESCRIPTION
**Pull request by cbillen**
support pass through multiple arguments in assignment operator otherwise with the use of mobility gem for instance with 1 arguments expected, received 2

### CODE REVIEW

File: lib/normalizr/concern.rb [hunk 1]
```diff
@@ -20,8 +20,8 @@ def normalize(*args, &block)
                 value = Normalizr.normalize(value, *options.before)
                 value = Normalizr.normalize(value, *options.after) if options.after.any?
               end
-
-              super(value)
+              args.insert(0, value)
+              super(*args)
```
Comment (dmeremyanin) [inline lib/normalizr/concern.rb:24]:
```suggestion
              super(value, *args)
```

I think we can pass the `value` variable as the 1st parameter to `super` without mutating the `args`.

File: lib/normalizr/concern.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def normalize(*args, &block)
 
         prepend Module.new {
           options.attributes.each do |method|
-            define_method :"#{method}=" do |value|
+            define_method :"#{method}=" do |value, *args|
```
Comment (dmeremyanin) [inline lib/normalizr/concern.rb:13]: Could you please change the name of the `args` variable here so it doesn't conflict with the `args` passed to the `normalize` method?

**Review by dmeremyanin — COMMENTED** [review]
Thanks for the PR! Just two minor comments/suggestions.

File: lib/normalizr/concern.rb (hunk 1 above)
Comment (cbillen) [reply lib/normalizr/concern.rb]: done

File: lib/normalizr/concern.rb (hunk 2 above)
Comment (cbillen) [reply lib/normalizr/concern.rb]: done

**Pull request merged**